Makefile

```
@@ -5,8 +5,11 @@ MAKEFLAGS += --no-builtin-variables
 COVERAGE_FILE ?= .coverage
 
 # allow overriding the name of the venv directory
-VENV_DIR ?= .venv
-UV_PROJECT_ENVIRONMENT=${VENV_DIR}
+DEFAULT_XDSL_VENV_DIR ?= .venv
+
+# use activated venv if any
+VENV_DIR ?= $(if $(VIRTUAL_ENV),$(VIRTUAL_ENV),$(DEFAULT_XDSL_VENV_DIR))
+export UV_PROJECT_ENVIRONMENT=${VENV_DIR}
```

Per https://github.com/xdslproject/xdsl/pull/3626/files , should this instead be the following?

```suggestion
UV_PROJECT_ENVIRONMENT=${VENV_DIR}
```

---

It's an explicit revert of that change, I don't understand why it was made, and it seems like it's required if we want uv to work for non `.venv` env names (https://github.com/astral-sh/uv/issues/9834#issuecomment-2539032524)